### DISCUSSION TITLE
Wrong markdown interpretation when using the sequence link, parenthesis, comma, word without whitespace

### DISCUSSION THREAD

**Discussion by lukaseder** [community/community#4285] [Projects and Issues]
When using this content in markdown: `blah blah (parenthesised blah and some link: https://github.com),oops`

Notice how the `,oops` is included in the link:

- blah blah (parenthesised blah and some link: https://github.com),oops

It wouldn't be if there was a space between `), oops`:

- blah blah (parenthesised blah and some link: https://github.com), oops

Here's a screenshot to illustrate the above:

> ![image](https://user-images.githubusercontent.com/734593/123227697-1fb4d780-d4d5-11eb-8078-1bcc016051cf.png)

PS: Great to see you now have a public issue tracker. Not sure if bug reports also belong here? Let me know if not.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
